cli/azd/extensions/azure.coding-agent/internal/cmd/coding_agent_config.go

```diff
@@ -21,12 +22,14 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
 	"github.com/azure/azure-dev/cli/azd/pkg/azdext"
 	"github.com/azure/azure-dev/cli/azd/pkg/entraid"
-	azdexec "github.com/azure/azure-dev/cli/azd/pkg/exec"
+	azd_exec "github.com/azure/azure-dev/cli/azd/pkg/exec"
 	"github.com/azure/azure-dev/cli/azd/pkg/graphsdk"
 	"github.com/azure/azure-dev/cli/azd/pkg/input"
 	"github.com/azure/azure-dev/cli/azd/pkg/output"
-	"github.com/azure/azure-dev/cli/azd/pkg/tools/github"
+	azd_git "github.com/azure/azure-dev/cli/azd/pkg/tools/git"
+	azd_github "github.com/azure/azure-dev/cli/azd/pkg/tools/github"
 	"github.com/azure/azure-dev/cli/azd/pkg/ux"
```

Comment:
oh, I though azd was going to surface all functionality thru a grpc azdClient. Is this consuming azd still as a library?